pkg/apis/specs/v1alpha2/tcp_route.go

```diff
@@ -0,0 +1,37 @@
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// TCPRoute is used to describe TCP inbound connections
+type TCPRoute struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	// Most recently observed status of the object.
+	// This data may not be up to date.
+	// Populated by the system.
+	// Read-only.
+	// More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status
+	// +optional
+	Status Status `json:"status,omitempty" protobuf:"bytes,2,opt,name=status"`
```

Comment:
Please remove this, status should contain a list of conditions but you've disabled the status sub-resource with `+genclient:noStatus`  so better remove it until the SMI spec has a well defined schema for status conditions.

Reply:
Thank you for the review. I did not realize this. I see that in the split spec, status is commented out. Would you prefer I follow that or just remove? Also, I see that status still exists in traffic target as well. Would you prefer I apply our decision that that also? Are you okay with me doing that work as a separate commit in this PR, or would you prefer a separate PR? Perhaps with the protobuf work you suggest below.

Reply:
I can remove those in a followup PR.